Tests/ParseSwiftTests/ParseCloudTests.swift

```diff
@@ -356,6 +356,28 @@ class ParseCloudTests: XCTestCase { // swiftlint:disable:this type_body_length
         }
     }
 
+    func testCustomError() {
+
+        let encoded: Data = "{\"error\":\"Error: Custom Error\",\"code\":2000}".data(using: .utf8)!
```

Comment:
Can you change this to a `guard` statement? Currently, we don't force unwrap in the SDK